cloudconfig/containerinit/container_userdata.go

```diff
@@ -380,3 +385,103 @@ else
 fi`[1:],
 		oldInterfacesFile, newInterfacesFile)
 }
+
+func populateNetworkInterfaces(networkFile string) string {
+	s := `
+if [ -f /usr/bin/python ]; then
+    python %s.py --interfaces_file %s
+else
+    python3 %s.py --interfaces_file %s
+fi
+`
+	return fmt.Sprintf(s, networkFile, networkFile, networkFile, networkFile)
+}
+
+func PopulateNetworkInterfacesScript() string {
+	s := `import subprocess, re, argparse, os, time
+from string import Formatter
+INTERFACES_FILE="/etc/network/interfaces"
```

Comment:
I'd put 1 blank line above this to separate out 'imports' from 'constants' for clarity.